pkg/metallb/addresspool.go

```diff
@@ -38,13 +33,14 @@ func NewIPAddressPoolBuilder(
 		"Initializing new IPAddressPool structure with the following params: %s, %s %s",
 		name, nsname, addrPool)
 
+	err := apiClient.AttachScheme(mlbtypes.AddToScheme)
+	if err != nil {
+		return nil
```

Comment:
I would add a glog in case this happens. It may have confusing behavior otherwise